src/monitor.cpp

```diff
@@ -49,7 +49,7 @@ Monitor::Monitor(Settings* settings_, MonitorManager* monman_, Rectangle rect_,
     , mouse { 0, 0 }
     , rect(rect_)
     , settings(settings_)
-    //, monman(monman_)
+    , monman(monman_)
```

Comment:
I know that the names are the wrong way round... actually the member variables should have the `_`-suffix, not the constructor arguments.